python/perf-kernels/rmsnorm.py

```diff
@@ -187,17 +187,23 @@ def parse_args():
 
     parser.add_argument('-d', "--dtype", default="fp16")
     parser.add_argument('-nb', "--no_benchmark", default=False, type=bool)
+    parser.add_argument("-v", action='store_true', default=False, help="Print out the best tuning config")
 
     return parser.parse_args()
 
 
 def main():
     args = parse_args()
+    global verbose
     if args.no_benchmark:
-        x = torch.randn(args.M_start, args.N_start)
+        x = torch.randn(args.M_start, args.N_start, device='cuda')
+        y = torch.zeros_like(x, device='cuda')
+        n_rows, n_cols = x.shape
+        blk_size = triton.next_power_of_2(n_cols)
```

Comment:
@xiaohuguo2023 Did you notice any big performance drop if blk_size >65k?

Reply:
yes, with blk_size>65k, start to have vgpr spills

Reply:
my next PR will address this issue